bagel/pheno_utils.py

```diff
@@ -18,11 +18,11 @@
 DICTIONARY_SCHEMA = dictionary_models.DataDictionary.schema()
 
 AGE_HEURISTICS = {
-    "float": NB.pf + ":float",
-    "int": NB.pf + ":int",
-    "euro": NB.pf + ":euro",
-    "bounded": NB.pf + ":bounded",
-    "iso8601": NB.pf + ":iso8601",
+    "FromFloat": NB.pf + ":FromFloat",
```

Comment:
Unless it breaks something, I suggest we keep the original simple dict keys `"float"`, `"int"`, etc as these are just meant to be internal aliases for the URIs and never enter the actual graph data files. This will make our lives easier down the line since they're shorter to type + means we don't need to update the functions that use this dict when/if we update the actual URI values.